api/v1alpha1/packagebundle_types.go

```diff
@@ -94,19 +94,6 @@ type VersionImages struct {
 	Digest string `json:"digest"`
 }
 
```

Comment:
Do we need a period of backward compatibility for this and just leve the version config until everything in build tooling uses schema?

Reply:
I think there will be a breaking change at some point, I'd rather it be before GA than after GA.

Reply:
I was thinking if you left the config parsing, it would not fail parsing that contains a config